GDAS-validation/gdas_config/config.atmanl

```diff
@@ -6,10 +6,9 @@
 echo "BEGIN: config.atmanl"
 
 export CASE_ANL=${CASE}
-#export CASE_ANL=C384
```

Comment:
@RussTreadon-NOAA Can we run C768 for atmanl now?

Reply:
Yes, we can run the minimizer on C768 for certain combinations of `[layout_x, layout_y]`.  `config.resources` in this PR sets `[layout_x, layout_y]` = `[8, 12]`.  I've run `fv3jedi_var.x` with `CASE_ANL=C768` using the `[8, 12]` layout.

Reply:
@RussTreadon-NOAA Thanks for your explanation.